sources/packages/web/src/views/FinancialInfo.vue

```diff
@@ -3,7 +3,10 @@
     <div class="p-card p-m-4">
       <Section title="More questions to come..." />
       <ContentGroup>
-        <Section :title="questions.previousYearTaxReturn.title">
+        <Section
+          type="secondary"
+          :title="questions.previousYearTaxReturn.title"
+        >
           <template #sub-title>
             <StringToHtmlParagraphs
```

Comment:
Missed to mention this in the review of previous PR, but please add a comment somewhere in the StringToHTMLParagraphs.vue to mention when should someone use this component and what will page look like when this used. Eg. "///Long text or String// wil look like <This>

Reply:
Added some comments.